lib/hx509/hx_locl.h

```diff
@@ -63,8 +63,8 @@
 #include <pkcs8_asn1.h>
 #include <pkcs9_asn1.h>
 #include <pkcs12_asn1.h>
-#include <ocsp_asn1.h>
-#include <pkcs10_asn1.h>
+#include <ocsp_asn1.hx>
```

Comment:
Why is this needed?

Reply:
The rest LGTM.

Reply:
It's really a dependency issue, I think the fix is this

```
diff --git a/lib/hx509/Makefile.am b/lib/hx509/Makefile.am
index b58deb3e3..09643c43a 100644
--- a/lib/hx509/Makefile.am
+++ b/lib/hx509/Makefile.am
@@ -164,7 +164,7 @@ hxtool-commands.c hxtool-commands.h: hxtool-commands.in $(SLC)
 dist_hxtool_SOURCES = hxtool.c
 nodist_hxtool_SOURCES = hxtool-commands.c hxtool-commands.h
 
-$(hxtool_OBJECTS): hxtool-commands.h hx509_err.h
+$(hxtool_OBJECTS): hxtool-commands.h $(nodist_include_HEADERS)
 
 hxtool_LDADD = \
        libhx509.la \
```